phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less

```diff
@@ -566,39 +566,39 @@
     -fx-background-image: url('/org/phoenicis/javafx/themes/breezeDark/icons/mainwindow/apps/all.png');
 }
 
-#customButton {
+#applications-custom-button {
```

Comment:
What about the other themes? Don't they require similar changes?

Reply:
They use the default icons from what I saw but you can check again.

Reply:
I think the standard theme requires similar changes:

https://github.com/PhoenicisOrg/phoenicis/blob/d470dc6ac00e79d629048ca03a7d62be5be84e4e/phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/standard/main.less#L868-L870

https://github.com/PhoenicisOrg/phoenicis/blob/d470dc6ac00e79d629048ca03a7d62be5be84e4e/phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/standard/main.less#L918-L921